### PR TITLE
Feat/pm 1065 players catalog

### PR DESCRIPTION
**Pull request by dejwi**
[PM-1065](https://playmakerpro.atlassian.net/browse/PM-1065)

### Task Description

- premerged footer
- players catalog
- pagination component
- button missing variant

### CODE REVIEW

File: components/catalogs/players/card/stats.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}: {
+  title: string
+  sub: string
+  icon: React.ReactElement
+  className?: string
+}) => (
+  <div
+    className={clsx(
+      className,
+      'flex flex-1 flex-col items-center justify-center sd:flex-row sd:gap-4',
```
Comment (mati-1) [inline components/catalogs/players/card/stats.tsx:57]:
```suggestion
      'flex flex-1 flex-col items-center justify-center md:flex-row md:gap-4',
```

File: components/catalogs/players/card/stats.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    )}
+  >
+    {React.cloneElement(icon, { className: 'icon-20 sd:icon-24 fill-base' })}
+    <div className="mt-2 flex flex-col sd:items-start ">
```
Comment (mati-1) [inline components/catalogs/players/card/stats.tsx:61]:
```suggestion
    <div className="mt-2 flex flex-col md:items-start ">
```

**Comment by mati-1** [comment]
image
mb add md: instead of sd: in the stats?

like that:
image

**Comment by mati-1** [comment]
i think it's really annoying when the click is in all the stats areas (not just the stats button).
image

in the figma it is suggested to click button?
image

File: components/catalogs/players/card/stats.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import CleatIcon from '@/icons/Cleat.svg'
+import ClockIcon from '@/icons/ClockVariant.svg'
+
+interface IProps {
+  golas: number
+  playedMinutes: number
+  playedMatches: number
+}
+
+const PLayerCardStats = ({ golas, playedMatches, playedMinutes }: IProps) => (
```
Comment (mati-1) [inline components/catalogs/players/card/stats.tsx:17]: golas? 😆

Comment (mati-1) [reply components/catalogs/players/card/stats.tsx:17]: and PLayer?

Comment (dejwi) [reply components/catalogs/players/card/stats.tsx:17]: 😨

**Review by mati-1 — APPROVED** [review]
everythin' is fine 👍

**Pull request merged**